hubblestack/utils/signing.py

```diff
@@ -146,8 +193,22 @@ def read_certs(*fnames):
                 with open(fname, 'r') as fh:
                     for i in split_certs(fh):
                         yield i
-            except Exception as exception_object:
-                log.error('error while reading "%s": %s', fname, exception_object)
+            except Exception as E:
+                log_level = log.debug
+                if check_verif_timestamp(fname) == True:
```

Comment:
Redundant comparison to True, check_verif_timestamp() returns bool value already.